data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1789 @@
+			choice
+				`	"I'll do what I can to help."`
+					goto leaders
+				`	"It sounds like you've contributed much to the Lunarium cause. How have you helped the Resistance?."`
+			`	"Yes, well observed. Kept the Lunarium Redoubt safe from prying Heliarch eyes and ears, my house has. Under me, assured that found by them, no dealings of ours - no dealings involving you - are," she says in a smug tone.`
+			label leaders
+			`	The Kimek who accompanied you here ask that you continue on. Lobmab and her bodyguards lead the way through busy cubicles and hallways, where dozens of Kimek work at computers and screens, as if monitoring the whole city. At the end of what seems to be the main passage of the complex, you pass through a doorway to find the Lunarium leaders seated around a table. You're shown to a seat, and as Lobmab comes in at the end, one of the bodyguards announces, "Lady Lobmab Bebliss, head of our great House."`
+			`	Some of the others at the table look to one another, and one of the Kimek who accompanied you all the way from your ship sighs quietly before coming to Lobmab. "Most welcome you are, Lady Lobmab," he says, as he begins pointing to the leaders one by one, beginning with the Kimek seated to your left. "Your host, Chiree is, the head of our local forces and organizer of many of our operations. Fought the Heliarchs since a young age, she has." He continues pointing to them in a clockwise pattern, now to the Arach to the left of Chiree. "Likewise, helping us for decades now, Tummug has, in spreading word of our group, and bringing to the people the truth about the Heliarchy. Another prominent Arach, and vital to the task soon to be discussed, Oobat is. A studious linguist, and aspiring diplomat, she is. Already worked together, I believe you and Pyakri have? One of our most senior militia officers, she is." He takes a pause, looking at Lobmab, almost as if waiting for some approval regarding his introductions of the others. She simply nods, and he moves on. "And... Elirom. A Heliarch Arbiter who defected over a century ago. Advising us since, he has." He waits for her to say something, before looking back at the table again and pointing to you. "Oh, of course. <first> <last>, the human who's helped us greatly, and who joining us today, is."`
```

Comment:
Is this assuming I've not met Chiree?
If I've done any of the Smuggling missions, I should recognise her, this doesn't require that I have actually got to Grenades, so I may just know her as a charity person, not a Lunarium member. I feel like there should be some reaction for that case. Perhaps `Resistance: Questions` should be offered if I've started that chain but haven't finished it (unless I failed part of it). That's going to be a chonky condition set.

Reply:
This is an admittedly awkward paragraph, but the point of it is so that I can introduce all of the Lunarium leaders without needing to account for every possible combination of the player having met them (met Chiree and Elirom, or Elirom and Pyakri, or Chiree, Pyakri and Tummug, and so on). That'd be very annoying to do, even if branches do technically make it doable, so I opted for just an exposition to someone other than the player (Lobmab) who hasn't met most of these people.
Basically, I wanted to avoid that "chonky condition set" you mention.

I'll edit this bit to avoid the "Kimek to your left" assuming you don't know her, would rather not have to do any part of the branching I've considered for this.